src/cclib/io/ccio.py

```diff
@@ -131,49 +138,17 @@ def guess_filetype(inputfile):
     return filetype
 
 
-def ccread(source, *args, **kargs):
-    """Attempt to open and read computational chemistry data from a file.
-
-    If the file is not appropriate for cclib parsers, a fallback mechanism
-    will try to recognize some common chemistry formats and read those using
-    the appropriate bridge such as OpenBabel.
-
-    Inputs:
-        source - a single logfile, a list of logfiles (for a single job),
-                 an input stream, or an URL pointing to a log file.
-        *args, **kargs - arguments and keyword arguments passed to ccopen
-    Returns:
-        a ccData object containing cclib data attributes
-    """
-
-    log = ccopen(source, *args, **kargs)
-    if log:
-        if kargs.get('verbose', None):
-            print('Identified logfile to be in %s format' % log.logname)
-        # If the input file is a CJSON file and not a standard compchemlog file
-        cjson_as_input = kargs.get("cjson", False)
-        if cjson_as_input:
-            return log.read_cjson()
-        else:
-            return log.parse()
-    else:
-        if kargs.get('verbose', None):
-            print('Attempting to use fallback mechanism to read file')
-        return fallback(source)
-
-
-def ccopen(source, *args, **kargs):
+def ccopen(source, *args, **kwargs):
     """Guess the identity of a particular log file and return an instance of it.
 
     Inputs:
         source - a single logfile, a list of logfiles (for a single job),
                  an input stream, or an URL pointing to a log file.
-        *args, **kargs - arguments and keyword arguments passed to filetype
+        *args, **kwargs - arguments and keyword arguments passed to filetype
 
-    Returns:
-      one of ADF, DALTON, GAMESS, GAMESS UK, Gaussian, Jaguar, Molpro, MOPAC,
-      NWChem, ORCA, Psi, QChem, CJSON or None (if it cannot figure it out or
-      the file does not exist).
+    Returns: one of ADF, DALTON, GAMESS, GAMESS UK, Gaussian, Jaguar, Molpro,
```

Comment:
In this format "returns" is usually on its own line. I think it's for easier parsing with sphinx.